.gitignore

```diff
@@ -23,4 +23,8 @@ coverage/
 /tmp
 
 conferences_data.rb
-conferences_raffle.txt
\ No newline at end of file
+conferences_raffle.txt
+
+Gemfile
```

Comment:
Putting the Gemfile in the .gitignore file is very bad practice — the Gemfile is the only way a team can track the additions of needed components, libraries, etc... If the Gemfile is added to .gitignore, anyone working on the project could add any gem they want and need to their own local file, and push changes that "work on their machine" but obviously wouldn't be working on anyone else's, because needed gems (i.e. libraries) are missing. 
If you absolutely need sqlite in your Gemfile (which you shouldn't, because we use postgres, not sqlite) you will have to comment it out any time you push new changes to the project, which is a lesser evil than removing the Gemfile altogether. I hope that helps!